Sources/App/Core/Extensions/SQLKit+functions.swift

```diff
@@ -49,44 +49,46 @@ func lower(_ arg: SQLExpression) -> SQLFunction {
     SQLFunction("LOWER", args: arg)
 }
 
+
 func unnest(_ array: SQLExpression) -> SQLFunction {
     SQLFunction("UNNEST", args: array)
 }
 
+
 func to_tsvector(_ array: SQLExpression) -> SQLFunction {
```

Comment:
Actually, we should probably fully "swiftify" this and make it `toTSVector` or `toTextSearchVector` or something.

Reply:
Although there's an argument to be made to keep it aligned with the underlying SQL function. OTOH `to_tsvector` with a weight which wraps a `to_tsvector` in `setweight` kind of breaks that convention already.